packages/desktop/views/dashboard/vesting/Vesting.svelte

```diff
@@ -56,10 +66,33 @@
                     : undefined,
         },
     ]
-
-    let modal: Modal
-    let timeUntilNextPayout = DEFAULT_EMPTY_VALUE_STRING
     $: $selectedAccountVestingPayouts, (timeUntilNextPayout = getTimeUntilNextPayout())
+    $: canCollect =
+        $selectedAccountVestingUnclaimedFunds === 0 ||
+        hasTransactionInProgress ||
+        (minRequiredStorageDeposit !== null &&
+            $selectedAccount?.balances?.baseCoin?.available < minRequiredStorageDeposit) ||
+        hasOutputsToConsolidate === false
```

Comment:
1. Can we change this line to just `hasOutputsToConsolidate`, instead of:
       
 ```
hasOutputsToConsolidate === false
            ? false
            : true
``````
2. These conditions and naming `canCollect` are a bit confusing:

`canCollect` should be `true` if :

- have funds to claim: $selectedAccountVestingUnclaimedFunds > 0
- there are no transactions in progress so: `!hasTransactionInProgress`
- if you have enough balance available to cover minRSD =>  `$selectedAccount?.balances?.baseCoin?.available > minRequiredStorageDeposit`
- and `hasOutputsToConsolidate === true` 

So to sum it up you can collect if there are funds to claim, no transactions in progress, have available funds to cover MRSD  and have outputs to consolidate.

Maybe I have misunderstood this :see_no_evil: